.env.example

```
@@ -59,5 +59,5 @@ RABBITMQ_TLS=false
 
 FEATURE_COMMON_SETTINGS_ENABLED=false
 FEATURE_COMMON_SETTINGS_QUEUE=chat.settings.updated.queue
-FEATURE_COMMON_SETTINGS_EXCHANGE="settings exchange"
+FEATURE_COMMON_SETTINGS_EXCHANGE="settings.exchange"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)